frontline-functions-quickstart/assets/providers/customers.private.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable camelcase */
+/**
+ * Generates a static list of customers based on the information provided through env vars
+ */
+const generateCustomersList = (context) => {
+  const customers = [];
+
+  if (
+    context.PHONE_NUMBER_FOR_CUSTOMER_1 &&
+    context.NAME_FOR_CUSTOMER_1 &&
+    context.AGENT_USERNAME
+  ) {
+    customers.push({
+      customer_id: customers.length,
+      display_name: context.NAME_FOR_CUSTOMER_1,
+      channels: [
+        { type: 'email', value: 'customer_test_1@example.com' },
+        { type: 'sms', value: context.PHONE_NUMBER_FOR_CUSTOMER_1 },
+        // { type: 'whatsapp', value: 'whatsapp:+12068664904' }, // TODO: figure out Whatsapp
+      ],
+      links: [
+        {
+          type: 'Facebook',
+          value: 'https://facebook.com',
+          display_name: 'Social Media Profile',
+        },
+      ],
+      worker: context.AGENT_USERNAME,
+    });
+  }
+
+  if (
+    context.PHONE_NUMBER_FOR_CUSTOMER_2 &&
+    context.NAME_FOR_CUSTOMER_2 &&
+    context.AGENT_USERNAME
+  ) {
+    customers.push({
+      customer_id: Math.floor(Math.random() * 1000),
```

Comment:
`customer_id` should not be dynamic, if you create a conversation and change the customer id, conversation will no longer be linked to the customer.

Reply:
Got it, makes sense, I'll just assign static numbers then.